src/components/modules/toolbar/inline.ts

```diff
@@ -279,70 +279,6 @@ export default class InlineToolbar extends Module<InlineToolbarNodes> {
     this.removeAllNodes();
   }
 
-  /**
-   * Returns inline toolbar settings for a particular tool
-   *
-   * @param tool - BlockTool object
-   * @returns {string[] | boolean} array of ordered tool names or false
-   */
-  private getInlineToolbarSettings(tool: BlockTool): string[] | boolean {
-    /**
-     * InlineToolbar property of a particular tool
-     */
-    const settingsForTool = tool.enabledInlineTools;
-
-    /**
-     * Whether to enable IT for a particular tool is the decision of the editor user.
-     * He can enable it by the inlineToolbar settings for this tool. To enable, he should pass true or strings[]
-     */
-    const enabledForTool = settingsForTool === true || Array.isArray(settingsForTool);
-
-    /**
-     * Disabled by user
```

Comment:
There were many useful comments describing all the cases. We need to move it to the new code.